hack/test/e2e-iso.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eou pipefail
+
+source ./hack/test/e2e.sh
+
+function cleanup() {
+  kill $(cat ${TMP}/qemu.pid); kill $(jobs -p) || true; rm -rf ${TMP}
+}
+
+mkdir -p ${TMP}
+
+qemu-img create -f qcow2 ${TMP}/disk.qcow2 4G
+
+qemu-system-x86_64 \
+  -m 2048 \
+  -curses \
+  -nographic \
+  -serial file:${TMP}/test.log \
+  -pidfile ${TMP}/qemu.pid \
+  -enable-kvm \
+  -cpu host \
+  -boot cd \
+  -netdev user,id=talos,ipv4=on,net=10.254.0.0/24,dhcpstart=10.254.0.10,hostfwd=tcp::50000-:50000,hostfwd=tcp::6443-:6443,hostname=control-plane-1 \
+  -device virtio-net,netdev=talos \
+  -cdrom ${ARTIFACTS}/talos-amd64.iso \
+  -hda ${TMP}/disk.qcow2 &
+
+trap 'cleanup' EXIT
+
+${TALOSCTL} gen config \
+  --additional-sans 127.0.0.1 \
+  --install-disk /dev/sda \
+  --install-image ghcr.io/talos-systems/installer:latest \
+  --output-dir ${TMP} \
+  test https://10.254.0.10:6443
+
+timeout -v --preserve-status 1m  bash -c "until ${TALOSCTL} apply-config -n 127.0.0.5 --insecure -f ${TMP}/init.yaml; do sleep 5; done"
```

Comment:
Just curious, why 127.0.0.5?

Reply:
typo ... not even sure how the hell that is working.